app/assets/javascripts/admin/addon/templates/emojis.hbs

```diff
@@ -59,4 +62,4 @@
       </tbody>
     </table>
   {{/if}}
-</div>
\ No newline at end of file
+</div>
```

Comment:
There's an end of file line here which is making the lint fail.

Reply:
Ahh thank you!! I was trying to figure it out and was stumped 😅